Custom-Audio-Driver/External-Audio-Device/OTDefaultAudioDevice.m

```diff
@@ -543,6 +552,11 @@ - (void) handleInterruptionEvent:(NSInteger) interruptionType
             case AVAudioSessionInterruptionTypeEnded:
             {
                 OT_AUDIO_DEBUG(@"AVAudioSessionInterruptionTypeEnded");
+                if (@available(iOS 15, *)) {
+                    [self setBluetoothAsPrefferedInputDevice];
+                } else {
+                    // do nothing refer comments for self.setBluetoothAsPrefferedInputDevice method.
```

Comment:
Same thing here.  How do you feel about logging this, instead, to make it more difficult to miss?

Reply:
We are making BT work the same as before, everywhere. There is no "missing" or "changed behavior" stuff to log.